crates/solver/src/main.rs

```diff
@@ -260,6 +260,18 @@ struct Arguments {
     /// The maximum number of settlements the driver considers per solver.
     #[structopt(long, env, default_value = "20")]
     max_settlements_per_solver: usize,
+
+    /// The Balancer V2 factories to consider for indexing liquidity. Allows
+    /// specific pool kinds to be disabled via configuration. Will use all
+    /// supported Balancer V2 factory kinds if not specified.
+    #[structopt(
+        long,
+        env,
+        possible_values = &BalancerFactoryKind::variants(),
+        case_insensitive = true,
+        use_delimiter = true
+    )]
```

Comment:
Do we plan to include a default here? or are we ok with changing our configurations for this right away?